src/jarallax.esm.js

```diff
@@ -64,9 +65,29 @@ domReady(() => {
 // list with all jarallax instances
 // need to render all in one scroll/resize event
 const jarallaxList = [];
+let globalParallaxUpdateInit = false;
 
 // Animate if changed window size or scrolled page
 let oldPageData = false;
+
+function parents(elem, selector) {
+    const elements = [];
+    const ishaveselector = selector !== undefined;
```

Comment:
`hasSelector` would be a better name here.

Reply:
Fixed